Description: Use x-terminal-emulator as terminal
Author: Joachim Breitner <nomeata@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2012-07-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: xmonad/XMonad/Config.hs
===================================================================
--- xmonad.orig/XMonad/Config.hs	2015-08-15 14:43:32.443704478 +0200
+++ xmonad/XMonad/Config.hs	2015-08-15 14:43:32.439704396 +0200
@@ -152,7 +152,7 @@
 -- | The preferred terminal program, which is used in a binding below and by
 -- certain contrib modules.
 terminal :: String
-terminal = "xterm"
+terminal = "x-terminal-emulator"
 
 -- | Whether focus follows the mouse pointer.
 focusFollowsMouse :: Bool
